### ISSUE TITLE
Docker compose install error with ModuleNotFoundError: No module named 'trino'

### ISSUE THREAD

**Issue by adongden** [apache/superset#26062]
I follow the doc to use docker compose to deploy superset on Linux, why I get this error:
<img width="978" alt="image" src="https://github.com/apache/superset/assets/122472234/a980eafa-1b0f-410a-810a-62281a00dbc7">
What I expect is that Docker Compose will automatically install these dependencies.

**Comment by shenrenguo** [comment]
add 'trino' package in docker/requirements-local.txt

**Comment by cusniwtt** [comment]
I cant find `docker/requirements-local.txt` but `local.txt` is in `requirements/local.txt`

**Comment by adongden** [comment]
<img width="386" alt="image" src="https://github.com/apache/superset/assets/122472234/7a99379a-94ed-46cd-8f40-622894809bf8">
After asking in Slack, suspect it's a BUG that can be solved by specifying version `3.0.1`. Waiting for an official reply to confirm if it's fixed.

**Comment by adongden** [comment]
> <img alt="image" width="386" src="https://user-images.githubusercontent.com/122472234/284802341-7a99379a-94ed-46cd-8f40-622894809bf8.png"> After asking in Slack, suspect it's a BUG that can be solved by specifying version `3.0.1`. Waiting for an official reply to confirm if it's fixed.

However, I must point out a new issue. If you have attempted installation on the `master` branch, please use the following script to remove the remaining content of the original image.
```shell
docker container prune
docker volume prune
docker volume rm superset_db_home 
```

**Comment by sfirke** [comment]
Fixed by #26066
